libs/exavmlib/lib/Range.ex

```diff
@@ -89,6 +89,39 @@ defmodule Range do
             "got: #{inspect(first)}..#{inspect(last)}"
   end
 
+  if Version.match?(System.version(), "~> 1.12") do
+    @doc """
+    Returns the size of `range`. Available Elixir 1.12 and later.
+
```

Comment:
Since this function will always appear in the documentation, we should add a note that this function is unavailable if the VM is built with an Elixir version prior to 1.12.

Reply:
Yeah, it states:
```Returns the size of `range`. Available Elixir 1.12 and later.```

But always open for textsmithing..

Reply:
Sorry, disregard that comment. I noticed that right before I pisted that comment, but my fat thumb must have hit comment instead of cancel.